reference.md

```diff
@@ -30,26 +30,39 @@ These are the properties that currently exist in the TPS repository, but they ar
 | [bricked](/properties/abilities/bricked.md) | The bricked ability represents whether the robot was bricked (unable to participate due to a technical issue) at any point in the match for any amount of time. | This property stores a boolean (true/false) value depending on whether or not the robot was bricked at any point in the match for any amount of time.<br><ul><li>true - the robot bricked</li><li>false - the robot did not brick</li></ul> |
 | [coopertition](/properties/abilities/coopertition.md) | If the robot's alliance met the coopertition requirements (regardless of whether or not the other alliance reciprocated). | This property stores a boolean (true/false) value |
 | [defense](/properties/abilities/defense.md) | The defense ability represents whether the robot played defensively at any point in the match for any amount of time. | This property stores a boolean (true/false) value depending on whether or not the robot played defensively at any point in the match.<br><ul><li>true - the robot played defensively</li><li>false - the robot never played defensively</li></ul> |
+| [endgame-cage-level-2025](/properties/abilities/endgame-cage-level-2025.md) | The endgame-cage-level-2025 ability represents the level of the cage climb of the robot at the end of the match. | This property stores an integer (0 - 3) based on the level of the cage climb.<br><ul><li>3: deep cage</li><li>2: shallow cage</li><li>1: parked</li><li>0: none</li></ul> |
```

Comment:
you don't need to worry about this file at all, github actions will automatically generate it when the pr gets merged (it's ok that you already made changes to it but don't worry about keeping it in sync any further as it'll automatically be taken care of)